src/scss/base.scss

```diff
@@ -189,6 +194,78 @@
 		@include oFormsGroupInlineRadioCheckbox($class);
 	}
 }
+/// @access public
+/// @param {string} $class - The block level class name (BEM naming convention).
+/// @output Styles for form status such as "saving" and "saved".
+@mixin oFormsStatusFeature($class: 'o-forms') {
+	// Icons align to a 40x40 pixel grid with 10px padding on each side.
+	// In this case however we want to remove that padding to align the icon flush with the input.
+	// https://github.com/Financial-Times/fticons/blob/master/contributing.md#design
+	$status-icon-size: $_o-forms-field-small-height;
+	$status-icon-padding: ($status-icon-size/40) * 10;
```

Comment:
What is the benefit over making this calculating v setting the variable to 10px?

Reply:
👍